components/rmrk/Gallery/Search/SearchBarCollection.vue

```diff
@@ -142,9 +142,9 @@ export default class CollectionSearchBar extends mixins(KeyboardEventsMixin) {
   @Emit('update:listed')
   @Debounce(50)
   updateListed(value: string | boolean): boolean {
-    const v = String(value)
+    const v = String(value) === 'true'
```

Comment:
could you change const name to fix deepsource error?

Reply:
done